plenum/server/consensus/view_change_service.py

```diff
@@ -287,7 +288,8 @@ def _finish_view_change(self):
 
         # Cancel View Change timeout task
         self._resend_inst_change_timer.stop()
-
+        # Unstash messages for new view
+        self._router.process_all_stashed(STASH_WAITING_NEW_VIEW)
```

Comment:
I think this one needs to be removed, since it's called in `process_new_view_checkpoints_applied` in Ordering service.

Reply:
removed